inc/3rd-party/hosting/presslabs.php

```diff
@@ -2,10 +2,6 @@
 
 defined( 'ABSPATH' ) || exit;
 
-if ( ! defined( 'PL_INSTANCE_REF' ) || ! class_exists( '\Presslabs\Cache\CacheHandler' ) ) {
-	return;
-}
-
 if ( ! file_exists( WP_CONTENT_DIR . '/advanced-cache.php' ) ) {
```

Comment:
@crystinutzaa What about this one? It was part of the entire flow of the file too. Right? Only load the file if this file also exists. I'm reading it correctly?

Reply:
Please recheck. I missed it